SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -129,6 +132,10 @@ namespace SpatialConstants
 	const Worker_ComponentId AUTHORITY_INTENT_COMPONENT_ID                  = 9980;
 	const Worker_ComponentId VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID        = 9979;
 
+	const Worker_ComponentId CLIENT_RPC_ENDPOINT_COMPONENT_ID				= 9978;
```

Comment:
Are these 3 new ids only used in `SchemaGenerator.cpp` currently? If possible, I'd prefer postponing the addition of these 3 ids till the next commit. Otherwise it is more error prone, since there might be a place, where we're using the old names (i.e. without `_LEGACY`), when the legacy ones are expected. However, if you're confident that these new names are not used anywhere except `SchemaGenerator.cpp` - I'm fine with leaving this in PR as is.

Reply:
This is the last commit.